rtl/eclrtl/eclrtl.cpp

```diff
@@ -3348,7 +3348,7 @@ bool rtlCodepageToCodepage(unsigned outlen, char * out, unsigned inlen, char con
     unsigned len = target - out;
     if(len < outlen)
         codepageBlankFill(outcodepage, target, outlen-len);
-    return U_SUCCESS(err);
+    return U_SUCCESS(err) != FALSE;
```

Comment:
icu strangely defines UBool to be int8 - which means visual studio complains about an implicit int->bool cast.